### PR TITLE
prometheus implement

### PR DESCRIPTION
**Pull request by tenisakb**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -44,6 +44,9 @@ WORKDIR $GOPATH/src/github.com/oysterprotocol/brokernode
 
 RUN go get -u github.com/gobuffalo/buffalo/buffalo
 
+# # Prometheus client:
+RUN go get -d -v github.com/prometheus/client_golang/prometheus
```
Comment (AaronVasquez) [inline Dockerfile]: Isn't this fetched from ` RUN go get -t -d -v ./...` below?

File: actions/app.go [hunk 2]
```diff
@@ -24,6 +25,18 @@ var app *buffalo.App
 // Visible for Unit Test
 var IotaWrapper = services.IotaWrapper
 var EthWrapper = services.EthWrapper
+var PrometheusWrapper = services.PrometheusWrapper
```
Comment (AaronVasquez) [inline actions/app.go]: Can we move this and the lines below to `services/prometheus.go`?

File: actions/transaction_genesis_hashes.go [hunk 3]
```diff
@@ -46,6 +47,12 @@ type transactionGenesisHashUpdateRes struct {
 
 // Creates a transaction.
 func (usr *TransactionGenesisHashResource) Create(c buffalo.Context) error {
+	start := time.Now()
+
+	defer func() {
```
Comment (AaronVasquez) [inline actions/transaction_genesis_hashes.go]: Does this need to be wrapped in a closure?

File: Dockerfile (hunk 1 above)
Comment (tenisakb) [reply Dockerfile]: I dont know what you mean, its needy to have it here to run prometheus.. how would you like to have it?

File: actions/transaction_genesis_hashes.go (hunk 3 above)
Comment (tenisakb) [reply actions/transaction_genesis_hashes.go]: Would you like to have here only function from service? In this function will be included histogram? But I dont know then how to work with defer function.

File: Dockerfile (hunk 1 above)
Comment (tenisakb) [reply Dockerfile]: I will take a look what lines below do exactly nad will try to remove

**Pull request closed without merging**